### PR TITLE
Feat/meta 3046 2

### PR DESCRIPTION
**Pull request by fatihdogmus**

### CODE REVIEW

File: repository/src/main/java/org/apache/atlas/repository/audit/ESBasedAuditRepository.java [hunk 1]
```diff
@@ -302,32 +295,14 @@ void createSession() throws AtlasException {
                 LOG.info("Updating ES total field limit");
                 updateFieldLimit();
             }
+            updateMappingsIfChanged();
```
Comment (DillionTan) [inline repository/src/main/java/org/apache/atlas/repository/audit/ESBasedAuditRepository.java:298]: what happens when we create the audit index successfully but fail to update the mappings? just curious why we don't update the mapping directly when we create the index?

File: repository/src/main/java/org/apache/atlas/repository/audit/ESBasedAuditRepository.java [hunk 2]
```diff
@@ -378,6 +363,47 @@ private void updateFieldLimit() {
...
+    }
+
+    private Response updateMappings(JsonNode indexInformationFromConfigurationFile) throws IOException {
+        Request request = new Request("PUT", INDEX_NAME + "/_mapping");
+        HttpEntity entity = new NStringEntity(indexInformationFromConfigurationFile.get("mappings").toString(), ContentType.APPLICATION_JSON);
+        request.setEntity(entity);
+        return lowLevelClient.performRequest(request);
+    }
+
+    private String getAuditIndexMappings() throws IOException {
```
Comment (mehtaanshul) [inline repository/src/main/java/org/apache/atlas/repository/audit/ESBasedAuditRepository.java:400]: Why make changes to an existing method?

File: repository/src/main/java/org/apache/atlas/repository/audit/ESBasedAuditRepository.java (hunk 1 above)
Comment (mehtaanshul) [reply repository/src/main/java/org/apache/atlas/repository/audit/ESBasedAuditRepository.java:298]: As per my understanding, new instances will have the correct mappings by default, this is to ensure mappings on existing instances get updated

**Comment by BurakAltas** [comment]
LGTM

**Review by DillionTan — APPROVED** [review]

**Pull request merged**